RELEASE_NOTES.md

```diff
@@ -331,6 +331,16 @@ You are strongly advised to take a backup first.
 - Canon EOS M200
 - Canon PowerShot G5 X Mark II
 - Canon PowerShot G7 X Mark III
+- Fujifilm GFX50S II (except lossy compressed)
```

Comment:
I'll merge, but this list is generally automatically generated (except for the camera handled by LibRAW of course).

Reply:
Ah, did not know that, sorry! In any case, thought it was more important to create the list of "suspended" cameras...